src/keyring/signature.rs

```diff
@@ -0,0 +1,22 @@
+//! Signing signature
+
+pub use ed25519_dalek as ed25519;
+pub use k256::ecdsa;
+
+/// Cryptographic signature used for block signing
+pub enum Signature {
+    ///  ED25519 signature
+    ED25519(ed25519::Signature),
+
+    /// ECDSA signagure (e.g secp256k1)
+    ECDSA(ecdsa::Signature),
```

Comment:
Please follow [RFC 430](https://rust-lang.github.io/api-guidelines/naming.html) naming conventions
```suggestion
    ///  ED25519 signature
    Ed25519(ed25519::Signature),

    /// ECDSA signagure (e.g secp256k1)
    Ecdsa(ecdsa::Signature),
```

Although it's debatable if an enum here is really helpful. It could just be a (newtype for) `Vec<u8>` unless the algorithm information is actually needed to be retained

Reply:
updated the naming to match RFC:
https://github.com/iqlusioninc/tmkms/pull/644/commits/7b8aaf00f1d1c43feced06db1251519ec6de6724

> Although it's debatable if an enum here is really helpful. It could just be a (newtype for) Vec<u8> unless the algorithm information is actually needed to be retained

I don't have strong opinion for either. Let me know if you think I should change it